lib/openstudio_measure_tester/runner.rb

```diff
@@ -32,6 +32,12 @@ def dashboard
       template.render
     end
 
+    def github_actions_report
+      gha_reporter = OpenStudioMeasureTester::GithubActionsReport.new(test_results_dir)
+      gha_reporter.make_minitest_step_summary_table
+      gha_reporter.make_minitest_annotations
+    end
```

Comment:
For now that's the two functions it calls, could be improved to include Rubocop + OpenStudio Style